extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/runtime/PanacheQueryImpl.java

```diff
@@ -1,10 +1,13 @@
 package io.quarkus.mongodb.panache.runtime;
 
+import java.beans.Introspector;
```

Comment:
I think we have something for that in some utils but I don't have my IDE handy, better not use `java.beans`.

Reply:
`io.quarkus.deployment.bean.JavaBeanUtil` but as it's inside the deployment module I cannot access it at runtime.
Anyway I will review the implementation next week

Reply:
I copied the code from `io.quarkus.deployment.bean.JavaBeanUtil`.
Mabe we need to move this to the core runtime module.